resources/assets/js/components/Application/Experience/ExperienceCommunityAccordion.tsx

```diff
@@ -0,0 +1,190 @@
+  titleBarDateRange,
+  baseExperienceMessages,
+} from "./BaseExperienceAccordion";
+import { Locales, getLocale } from "../../../helpers/localize";
+import { readableDate } from "../../../helpers/dates";
+
+interface ExperienceCommunityAccordionProps {
+  title: string;
+  group: string;
+  project: string;
+  startDate: Date;
+  endDate: Date;
+  isActive: boolean;
+  relevantSkills: ExperienceSkill[];
+  irrelevantSkillCount: number;
+  isEducationJustification: boolean;
+  showSkillDetails: boolean;
+  showButtons: boolean;
+  handleDelete: () => void;
+  handleEdit: () => void;
+}
+
+const experienceCommunityDetails = ({
+  locale,
+  intl,
+  title,
+  group,
+  project,
+  startDate,
+  endDate,
+  isActive,
+}: {
+  locale: Locales;
+  intl: IntlShape;
+  title: string;
+  group: string;
+  project: string;
+  startDate: Date;
+  endDate: Date;
+  isActive: boolean;
+}): React.ReactElement => {
+  const notApplicable = (
+    <p data-c-color="gray">
+      {intl.formatMessage(baseExperienceMessages.notApplicable)}
+    </p>
+  );
+  const endDateOrNa = endDate ? (
+    <p>{readableDate(locale, endDate)}</p>
+  ) : (
+    notApplicable
+  );
+  return (
+    <>
+      <div data-c-grid-item="base(1of2) tl(1of3)">
+        <p data-c-font-weight="bold">
+          {intl.formatMessage(baseExperienceMessages.experienceTypeLabel)}
+        </p>
+        <p>
+          <i
+            className="fas fa-people-carry"
+            data-c-color="c1"
+            data-c-margin="right(.25)"
+          />
+          <FormattedMessage
+            id="experienceCommunityAccordion.experienceTypeTitle"
+            defaultMessage="Community Experience"
+          />
+        </p>
+      </div>
+      <div data-c-grid-item="base(1of2) tl(1of3)">
+        <p data-c-font-weight="bold">
+          <FormattedMessage
+            id="experienceCommunityAccordion.roleLabel"
+            defaultMessage="Role / Job Title:"
+          />
+          {title ? <p>{title}</p> : notApplicable}
+        </p>
+      </div>
+      <div data-c-grid-item="base(1of2) tl(1of3)">
+        <p data-c-font-weight="bold">
+          <FormattedMessage
+            id="experienceCommunityAccordion.organizationLabel"
+            defaultMessage="Group / Organization / Community:"
+          />
+        </p>
+        {group ? <p>{group}</p> : notApplicable}
+      </div>
+      <div data-c-grid-item="base(1of2) tl(1of3)">
+        <p data-c-font-weight="bold">
+          <FormattedMessage
+            id="experienceCommunityAccordion.projectLabel"
+            defaultMessage="Project / Product:"
+          />
+        </p>
+        {project ? <p>{project}</p> : notApplicable}
+      </div>
+      <div data-c-grid-item="base(1of2) tl(1of3)">
+        <p data-c-font-weight="bold">
+          <FormattedMessage
+            id="experienceCommunityAccordion.statusLabel"
+            defaultMessage="Status:"
+          />
+        </p>
+        {status ? <p>{status}</p> : notApplicable}
```

Comment:
Should this be a prop? eslint is throwing a `no-restricted-globals` error here and I'm not sure where `status` is coming from.

Reply:
Actually, I don't think a status section is supposed to be in this accordion at all. Good catch!

Reply:
I took another look, I'm missing a few other fields from Josh's templates.